src/features/header/ui/UserMenu.tsx

```diff
@@ -0,0 +1,128 @@
+'use client'
+
+import { useEffect, useRef, useState } from 'react'
+
+import { signOut } from '@/features/auth/serverActions/signOut'
+import LoadingOverlay from '@/common/ui/LoadingOverlay'
+import { cn } from '@/common/utils/style/cn'
+import { Key, Menu, MenuItem } from 'react-aria-components'
+
+const PADDING_RIGHT = 'pr-6'
```

Comment:
Can just have one padding value `const PADDING = 'px-6'`, right?

Reply:
yeah... i had separated them due to a previous JSX strategy, but agreed will consolidate